internal/cmd/skupper/site/nonkube/site_create.go

```diff
@@ -79,7 +78,6 @@ func (cmd *CmdSiteCreate) InputToOptions() {
 		}
 
 		cmd.linkAccessEnabled = true
-		cmd.bindHost = "0.0.0.0"
```

Comment:
I believe that this must also be done to `site generate`.

Reply:
fixed, thanks.